project_silentdata.md

```diff
@@ -0,0 +1,188 @@
+﻿# Silent Data Polkadot Integration 
+ 
+ **Team Name**: Applied Blockchain Ltd
+
+**Payment Address**: USDC (Ethereum) address: 0x91a5ade2522ac8c3761922a4255e0fef89116a37
+
+**[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):**(https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2 
+
+## Project Overview :page_facing_up:
+
+This application is in response to an RFP 
+
+### Overview
+
+Applied Blockchain has developed Silent Data as a platform for proving properties of private off-chain (web2) data in blockchain smart contract (web3) applications (dApps).
+
+Silent Data leverages hardware secure enclaves with attestation, in particular, Intel SGX in order to enable privacy-preserving retrieval and processing of off-chain data, and generation of cryptographic proofs that are verifiable in blockchain smart contracts. This ensures that sensitive information is never revealed, not even to those hosting the platform, and code attestation with a smart contract link ensures that the code used to retrieve the data and generate the proofs cannot be modified or interfered with by the operator.
+
+Silent Data proof certificates are powerful because they can verifiably demonstrate that private data meets certain requirements without the need to reveal that data, and without that data being accessible to Silent Data operators. The service is initially centrally hosted, and will be decentralised over time in order so that it cannot be censored by the operator.

```

Comment:
Can you explain how such a service can be decentralised? What would that look like?

Reply:
Hi Semuelle,
One of the properties of SGX is that it is possible to prove what code is being run inside a particular enclave. This means that we would be able to provide decentralised "nodes" with prebuilt enclaves that contain the code required to run the Silent Data service and then provision them with encrypted secret keys so that they can create compatible proof certificates. Once provisioned, these nodes would be able to run in a decentralised fashion, independent from the original operator.